beast/observationmodel/observations.py

```diff
@@ -125,35 +99,33 @@ def setFilters(self, filters):
 
     def getMags(self, num, filters):
         raise Exception('Do not use as magnitudes')
-        return np.array([ self.data[tt][num] - self.distanceModulus for tt in filters])
```

Comment:
The removal of distanceModulus seems like it will affect the comparison of obs and model fluxes -- are their any sanity checks that all code is abiding by new convention?   i.e. -- when trimming the grid I get an error stating: "no models are brighter than the minimum ASTs run", which could be caused by mismatch in conventions.

Reply:
Given @lcjohnso's comment and error message, it's important to check that properly. @lcjohnso  can you give more to @karllark  to chew?

I agree with @karllark, Beast makes the models at the distance of the population, not the way around (in principle). The only place where the distance modulus in `Observations` class is useful is to generate mock data, but that could be left to the tester.

Reply:
Thanks for the further explanation.  If I'm understanding correctly, are you saying that the getMags and getFlux functions defined here are never used?  If so, then I'd advocate for removing them completely to avoid confusion rather than change their utility/purpose (from distance-corrected to non-distance-corrected).

Can you confirm that these functions are not used elsewhere?  Otherwise the abrupt change of removing the distance correction continues to worry me - shouldn't this removal require subsequent edits elsewhere to account for the different treatment of distance?

Reply:
The getmags is only used in observations.py itself. 
The getflux is used in other places, but the key is that the stock getflux function is redefined in datamodel.py.  And we have been redefining it this way for a long time.  This actually illustrates how having the distance in the stock versions in observations.py is dangerous.  The "experts" know not to use it, but newer users would not necessarily know this.  Removing it in the stock functions means that the experts and soon-to-be experts will both use the right functions.

Reply:
@lcjohnso in your first comment, you had an error.  Was this with the phat_small example?  I've just run this example and am not getting that error.

Reply:
Thanks -- I agree that your edits make good sense.  Sorry for the delay -- I didn't realize that `getFlux()` was being redefined in datamodel.py, so the non-distanceModulus version of the transformation was already in effect.

The error I was encountering was due to an unrelated issue.  I was testing the code on SMIDGE data where I'm only fitting a subset of the bands, and the `n_detected` keyword in `trim_grid.trim_models()` needed to be set to a non-default value to run correctly.